definitions/pipelines/immuno.cwl

```diff
@@ -305,6 +309,21 @@ outputs:
     fusion_evidence:
         type: File
         outputSource: rnaseq/fusion_evidence
+    unfiltered_fusion_seqs:
+        type: File
+        outputSource: rnaseq/unfiltered_fusion_seqs
+    unfiltered_fusions_json:
+        type: File
+        outputSource: rnaseq/unfiltered_fusions_json
+    filtered_fusion_seqs:
+        type: File
+        outputSource: rnaseq/filtered_fusion_seqs
+    filtered_fusions_json:
+        type: File
+        outputSource: rnaseq/filtered_fusions_json
+    parsed_fusion_calls:
+        type: File
+        outputSource: rnaseq/parsed_fusion_calls
```

Comment:
1) naming making it clear which of these files is the final result would be nice.  (a naive user is not going to know whether filtered or parsed is the end stage).   

2) If this PR is needed soon, then this is not required, but this seems like as good a place as any to start putting things in subfolders?  A gene_fusions_pizzly folder containing these results would take a few files out of the already overwhelming list.